### DISCUSSION TITLE
個人情報が乗っているため早急に削除を依頼

### DISCUSSION THREAD

**Discussion by Hzva** [edginer/eddiner#34] [General]
削除を依頼
http://bbs.eddibb.cc/test/read.cgi/liveedge/1709297315/

**Comment by edginer** [comment]
いずれのスレにおいても個人情報に該当する書き込みはないように見受けられます
対象のスレッドに対しこの事由で削除を行いたい場合は、対象のレス番号を示すかスレ全体がなぜ個人情報にあたるのかを明確に記述してください

**Comment by edginer** [comment]
対象スレッドいずれも削除させていただきました
なおbbs.eddibb.ccは必至チェッカーを現在提供しておらず、仮に言及されている必死チェッカーが外部のサービスである場合、そちらの削除等に関しては対応できかねますのでご容赦ください

**Reply by edginer** [reply]
なお本件ディスカッションに対しては回答がない場合、1週間後には削除させていただきます

**Reply by Hzva** [reply]
本当にありがとうございました。感謝してもしきれません。素早い対応に感謝します。
先程の説明は他の方に見られると困るので消しておきます。対応有難うございました。
